src/Controller/SettingsController.php

```diff
@@ -95,4 +92,9 @@ private function getForm(SettingsInterface $settings): FormInterface
             ],
         );
     }
+
+    private function cleanCache(SettingsInterface $settings): void
```

Comment:
Why a dedicated function since coule just run the single statement above in the script ?